UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/appsettings.json

```diff
@@ -26,14 +26,17 @@
   "KeyVaultSettings": {
     "ServiceUri": ""
   },
-    "FileShareService": {
-        "BaseUrl": "",
-        "BusinessUnit": "",
-        "ProductType": "",
-        "FssClientId": "",
-        "PageSize": "100",
-        "Start": "0"
-    },
+  "FileShareService": {
+    "BaseUrl": "",
+    "BusinessUnit": "",
+    "ProductType": "",
+    "FssClientId": "",
+    "PageSize": "100",
+    "Start": "0"
+  },
   "APPINSIGHTS_INSTRUMENTATIONKEY": "",
-  "AllowedHosts": "*"
-}
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "RadioNavigationalWarningsContext": ""
```

Comment:
No need to add connection string setting in appsettings.json. Please remove below config.
 "ConnectionStrings": {
    "RadioNavigationalWarningsContext": ""
}

Reply:
done required changes